README.md

```diff
@@ -1,2 +1,53 @@
-# Leaderboard
-This is a Repository where I will manage leaderboard having name and score of a person.
+# To-Do List
+
+>In this project I have setup an application that will consume API to create a game and save and fetch players's scores and built with Webpack. 
```

Comment:
- [ ] Let's try to be thorough in our documentation. This is the `Leaderboard` project, but your README contains information describing the `To-Do List`. I suggest you correct the title of your project. 👍🏻